config/checkstyle_checks.xml

```diff
@@ -116,6 +116,14 @@
     <property name="maximum" value="0"/>
     <property name="message" value="Trailing whitespace is not allowed"/>
   </module>
+  <!-- This is needed for correct metadata generation -->
+  <module name="RegexpSingleline">
+    <property name="id" value="propertyTypeOnNewLine"/>
```

Comment:
So we aren't going to fix the bug in the tool?

Reply:
I am not sure it is a bug in tool.
Generator expects this data to be on new line. Now we have a way to check this automatically and avoid such issues in future.

Reply:
It is a bug if the tool expects it and it doesn't give the user an error that something is wrong when it is given something else. Without documentation, and something to hand hold the users through this, it is completely unclear that this is "required". This has come up atleast twice since I have seen the tool's implementation and both times no one understood what the problem was or how to resolve it.

Reply:
I changed tool code to fail in case type or default value of check's property is missing/misplaced. Now it will throw an exception with clear message.

Reply:
I am ok with such fix.
@strkkk , thanks a lot for fixing this.